docker-compose.yml

```diff
@@ -38,5 +38,17 @@ services:
       - SCHEDULE_EXPRESSION_DELETE-EXPIRED-OTP=0 0 3 * * *
       - HMPPS_SQS_LOCALSTACK-URL=http://localstack:4566
 
+  clamav:
```

Comment:
This should be in either docker-compose.yml or docker-compose-local.yml so probably worth removing it from this file.